persistence/pre_persistence/genesis.go

```diff
@@ -180,19 +191,19 @@ func InitGenesis(u *PrePersistenceContext, state *GenesisState) error {
 		}
 	}
 	for _, validator := range state.Validators {
-		err := u.InsertValidator(validator.Address, validator.PublicKey, validator.Output, false, 2, validator.ServiceURL, validator.StakedTokens, 0, 0)
+		err := u.InsertValidator(validator.Address, validator.PublicKey, validator.Output, false, 2, validator.ServiceUrl, validator.StakedTokens, 0, 0)
```

Comment:
Minor NIT, I prefer and think is more readable, the following multi-line style:
```suggestion
		err := u.InsertValidator(
		   validator.Address,
		   validator.PublicKey,
		   validator.Output,
		   false,
		   2,
		   validator.ServiceUrl,
		   validator.StakedTokens,
		   0,
		   0,
		 )
```

The value arguments `2` and the two `0` are ambigious, can use comments in this case.

Reply:
I think that this argument list is too long too, can be replaced with a struct